experimental/Geometry/Multiindices.jl

```diff
@@ -0,0 +1,221 @@
+module Multiindices
+
+export OrderedMultiindex
+export linear_index
+export index_signature
+export HomogMultiindex
+export power
+export getindex
+
+
+abstract type Multiindex end
+
+mutable struct OrderedMultiindex <: Multiindex 
+  n::Int
+  p::Int
+  index::Vector{Int}
+  function OrderedMultiindex(n::Int, p::Int)
+    v = Vector{Int}(undef,p) 
+    for i in 1:p
+      v[i] = i
+    end
+    #@show n, p, v
+    return new( n, p, v )
+  end
+end
+
+function Base.show( io::Base.TTY, i::OrderedMultiindex ) 
+  outstr= "0"
+  for k in 1:length(i.index) 
+    outstr = outstr * " < " * string(i.index[k]) 
+  end
```

Comment:
```suggestion
  for k in i.index
    outstr *= " < $k"
  end
```

Alternatively you could use `join(i.index, " < ")`

Reply:
This operator exists in Julia? I thought, it didn't. If yes, I'll be happy to apply it.